js/request.js

```diff
@@ -150,7 +150,6 @@ exports.request = function(req, res, opts, handler) {
       var terminationMessage = 'Terminating server at http://[localhost]:' + opts.port + ' for ' + opts['module-path'] + ' / ' + opts.handler;
       res.end(terminationMessage + '\n');
       console.info(terminationMessage);
-      server.close();
```

Comment:
The point with terminating it via DELETE was to terminate it vaguely nicely, @proc.kill is a much harder termination.

Normally, one wants to terminate a program by falling off the end of `main` function as that allows any resources to be cleaned up.